frontend/src/libs/components/map/map.tsx

```diff
@@ -0,0 +1,43 @@
+import { Status, Wrapper } from '@googlemaps/react-wrapper';
+
+import { useCallback } from '~/libs/hooks/hooks.js';
+import { config } from '~/libs/packages/config/config.js';
+
+import { Spinner } from '../components.js';
+import { MapInnerComponent } from './map-inner-component/map-inner-component.js';
+
+const apiMapKey = config.ENV.API.GOOGLE_MAPS_API_KEY;
+
+type MapType = {
```

Comment:
Since you are not using it anywhere except component props lets remove it and move everything to the properties